bats/fb-content-katello.bats

```diff
@@ -140,12 +140,21 @@ EOF
   subscription-manager repos --enable="${ORGANIZATION_LABEL}_${PRODUCT_LABEL}_${YUM_REPOSITORY_LABEL}" | grep -q "is enabled for this system"
 }
 
-@test "install katello-agent" {
-  tPackageInstall katello-agent && tPackageExists katello-agent
+@test "install katello-host-tools" {
+  tPackageInstall katello-host-tools && tPackageExists katello-host-tools
+}
+
+@test "install package locally" {
+  run yum -y remove walrus
```

Comment:
Hrmm, another reason I thought to choose run was my assumptions around how the line below this would work:

```
tPackageInstall walrus-0.71 && tPackageExists walrus-0.71
```

In that, this should assert that 0.71 is installed and if not error out. I was not intending to test that the package could be installed but rather that at this point, walrus-0.71 is on the system, to allow for there to be applicable errata. So I think I assumed that `tPackageExists` would error if some other version of walrus was installed.